task-attribution.bs

```diff
@@ -0,0 +1,156 @@
+<pre class=metadata>
+Title: Task Attribution
+Status: CG-DRAFT
+Shortname:task-attribution
+Group: wicg 
+Level: none
+Editor: Yoav Weiss, Google https://google.com, yoav@yoav.ws, w3cid 58673
+
+URL: https://wicg.github.io/soft-navigation/task-attribution.html
+Repository: https://github.com/WICG/soft-navigation
+Test Suite: https://github.com/web-platform-tests/wpt/tree/master/soft-navigation-heuristics
+Abstract: This document defines infrastructure to enable browsers to know which tasks are ancestors for other tasks.
+Default Highlight: js
+Complain About: accidental-2119 yes
+</pre>
+
+<pre class=anchors>
+urlPrefix: https://html.spec.whatwg.org/C; spec: HTML;
+    type: dfn; url: #execute-the-script-element; text: execute the script element;
+</pre>
+<pre class=link-defaults>
+</pre>
+
+Introduction {#sec-intro}
+=====================
+
+<div class="non-normative">
+
+<em>This section is non-normative.</em>
+
+The user agent’s event loop is continuously running tasks, as well as microtasks.
+Being able to keep track of which task initiated which can be valuable in multiple cases:
+  * Enable user agents to create heuristics that rely on causal link between one operation (e.g. a user initiated click event) and another (e,g. a DOM modification).
+  * Enable user agents to make prioritization (of tasks as well as resource loading) "inheritable", and e.g. ensure that low-priority scripts cannot queue high-priority tasks.
+  * Enable causal user activation delegaion.
+  * Enable accumulating knowledge of resource loading dependency chains, and enable developers to draw insights from them.
+
+This specification outlines a mechanism that would enable user agents to deduce that information, and use it in various other algorithms that might be used for web exposed features.
+However, this specification does not define a web exposed feature in and of itself.
+
+</div>
+ISSUE: Should we discuss potential conncetion with AsyncContext and its use cases?
```

Comment:
I believe `conncetion` is a typo or not?

Reply:
It is!! thanks!